### PR TITLE
Update mqtt.markdown

### PR DESCRIPTION
**Pull request by martin3000**
Beginners could think is for configuring the broker itself. Clarified wording.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/mqtt.markdown [hunk 1]
```diff
@@ -67,7 +67,7 @@ There are [at least two](https://issues.apache.org/jira/browse/AMQ-6360) [issues
 
 The Mosquitto project runs a [public broker](https://test.mosquitto.org). This is the easiest to set up, but there is no privacy as all messages are public. Use this only for testing purposes and not for real tracking of your devices or controlling your home. To use the public mosquitto broker, configure the MQTT integration to connect to broker `test.mosquitto.org` on port 1883 or 8883.
 
-## Broker configuration
+## Configuration of the connection to the broker
```
Comment (jbouwh) [inline source/_integrations/mqtt.markdown:70]: I rather prefer not to update the headers if this is not strictly needed as with it also the page anchors change.
To specific this header it seems clear enough to me in it current form.

File: source/_integrations/mqtt.markdown [hunk 2]
```diff
@@ -83,7 +83,7 @@ If you experience an error message like `Failed to connect due to exception: [SS
 
 </div>
 
-### Advanced broker configuration
+### Advanced configuration of the connection to the broker
```
Comment (jbouwh) [inline source/_integrations/mqtt.markdown:86]: Same here. You could suggest a change the paragraph text though.

**Review by jbouwh — CHANGES_REQUESTED** [review]

**Comment by martin3000** [comment]
In the whole text, it is not clearly differentiated between the MQTT client integration and the broker. For most users this should be very confusing.

**Comment by jbouwh** [comment]
> In the whole text, it is not clearly differentiated between the MQTT client integration and the broker. For most users this should be very confusing.

I do not agree on this. The documentation is clear that the MQTT integration requires to choose a broker and tells about the alternatives. It is clear the documentation is not about setting up the broker you choose MQTT broker.

**Comment by frenck** [comment]
I think I'm with @jbouwh on this one, and therefore going to close this PR.

Nevetheless, thanks for being willing to contribute 👍 

../Frenck

**Pull request closed without merging**